misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/comparator/DumbNodeListComparator.java

```diff
@@ -39,9 +40,9 @@
  */
 public class DumbNodeListComparator<T extends Node> implements Comparator {
 
-    protected final List<T> newNodesList;
-    protected final List<T> oldNodesList;
-    protected String nodeKindName;
+    private final List<T> newNodesList;
+    private final List<T> oldNodesList;
```

Comment:
In some places, lists are pluralized, and here `List` prefix is used, wouldn't it be better to have `newNodes`, `oldNodes` for consistency?